packages/vite-app-ts/src/styles/css/app.css

```diff
@@ -1,3 +1,5 @@
+@import 'eth-components/ant/index.css';
```

Comment:
i don't see any file in the other pr about index.css?

Reply:
It is generated in the build so as to contain the css from Address.css and Balance.css